### DISCUSSION TITLE
Why Do Bundle Files Have Specific Size Limits in Bootstrap?

### DISCUSSION THREAD

**Discussion by MohamadSalman11** [twbs/bootstrap#40891] [Q&A] [answered]
Bootstrap sets size limits for files like bootstrap-grid.css (e.g., 6.5KB). I know it helps with performance ⚡ and smaller downloads 📦, but is there any specific reason for these limits? 🤔 And if the file needs to get bigger in the future, what’s the best way to handle that? 🔧 I'd love to hear your thoughts! 💬

**Comment by julien-deramond** [comment, marked as answer]
I wasn’t part of the core team when it was initially implemented, but from my perspective, it serves as a security measure that gets checked for every PR. This not only helps prevent mistakes but also acts, for me personally, as a reminder that every addition comes with a cost and should be minimized, especially for eco-design reasons.

> And if the file needs to get bigger in the future, what’s the best way to handle that?

If the file needs to grow, it should be a considered decision, but it’s not an issue. We can modify the limit in the same PR that updates or adds content. The rule is to increase the value by increments of 0.25.
